src/components/EChimpCards/EChimpCard.tsx

```diff
@@ -84,6 +86,12 @@ const EChimpCard = ({
             <p>{crStatus}</p>
           </div>
         )}
+        {status && (
+          <div className="echimp-card__cr-status">
```

Comment:
nitpick: Could maybe update this to something like `echimp-card__status` to more generically reflect that it works for statuses for both CRs and TDLs